packages/bits/src/lib/expander/expander.component.html

```diff
@@ -26,9 +26,9 @@
         </div>
     </div>
     <div class="nui-expander__body"
-         [class.hide-left-border]="hideLeftBorder"
-         [@expandedState]="state"
-         (@expandedState.done)="expandedStateDone()">
-        <ng-content *ngIf="renderBody"></ng-content>
```

Comment:
I noticed that the `renderBody` property still exists in the component. Do we need to keep it around, or can we get rid of it?